emmet-core/emmet/core/ml_potential.py

```diff
@@ -0,0 +1,124 @@
+from typing import TYPE_CHECKING, Dict, List, Union
+
+from emmet.core.material_property import PropertyDoc
+from matcalc.elasticity import ElasticityCalc
+from matcalc.eos import EOSCalc
+from matcalc.phonon import PhononCalc
+from matcalc.relaxation import RelaxCalc
+from pydantic import Field
+from pymatgen.analysis.elasticity import ElasticTensor
+from pymatgen.core import Structure
+
+if TYPE_CHECKING:
+    from ase.calculators.calculator import Calculator
+
+
+class MLIPDoc(PropertyDoc):
+    """Document model for matcalc-generated material properties from machine learning
+    interatomic potential predictions.
+
+    Includes
+    - relaxation properties
+        - final structure: relaxed pymatgen Structure object
+        - energy (float): final energy in eV
+        - volume (float): final volume in Angstrom^3
+        - lattice parameters: a, b, c, alpha, beta, gamma
+    - equation of state properties
+        - EOS (dict[str, list[float]]): with keys energies and volumes
+        - bulk modulus (float): Birch-Murnaghan bulk modulus in GPa
+    - phonon properties
+        - temperature (list[float]): temperatures in K
+        - free energy (list[float]): Helmholtz energies at those temperatures in eV
+        - entropy (list[float]): entropies at those temperatures in eV/K
+        - heat capacities (list[float]): heat capacities at constant volume in eV/K
+    - elasticity properties
+        - elastic tensor (ElasticTensor): pymatgen ElasticTensor object
+        - shear modulus (float): Voigt-Reuss-Hill shear modulus
+        - bulk modulus (float): Voigt-Reuss-Hill bulk modulus
+        - Young's modulus (float): Young's modulus
+    """
+
+    property_name = "mlip"
+
+    # relaxation attributes
+    final_structure: Structure = Field(description="ML-potential-relaxed structure")
+    energy: float = Field(description="Final energy in eV")
+    volume: float = Field(description="Final volume in Angstrom^3")
+    a: float = Field(description="Lattice length a in Angstrom")
+    b: float = Field(description="Lattice length b in Angstrom")
+    c: float = Field(description="Lattice length c in Angstrom")
+    alpha: float = Field(description="Lattice angle alpha in degrees")
+    beta: float = Field(description="Lattice angle beta in degrees")
+    gamma: float = Field(description="Lattice angle gamma in degrees")
+
+    # equation of state attributes
+    EOS: Dict[str, List[float]] = Field(
```

Comment:
I think we should keep every key lower case.

Reply:
Agreed.